packages/account/src/index.ts

```diff
@@ -16,7 +16,7 @@ export {
   type FeeQuotesOrDataResponse,
   createPaymaster,
 } from "@biconomy/paymaster";
-export { EthersSigner, convertSigner, type LightSigner } from "@biconomy/common";
+export { EthersSigner, convertSigner, type LightSigner, type SupportedSigner } from "@biconomy/common";
```

Comment:
what is this change for? does it have to be part of this PR purpose?

Reply:
No it's not relevant to this PR. I noticed we'll need to export this if the version of viem that we bundle gets dated - users will need to cast their signer. The long term solution for this is to move viem to a peer dep.